src/components/biggive-main-menu/biggive-main-menu.tsx

```diff
@@ -64,6 +64,9 @@ export class BiggiveMainMenu {
   }
 
   componentDidRender() {
+    document.body.style.paddingTop = this.host.offsetHeight + 'px';
+    this.host.classList.add('fixed');
+
```

Comment:
This seems to break the layout on the angular site. Screenshots from angular with the current `main` components and this branch:

![image](https://user-images.githubusercontent.com/159481/222717521-f17277e0-25c9-4bee-a04f-95c484ce5ac3.png)
![image](https://user-images.githubusercontent.com/159481/222717703-b152b0d5-e992-46c7-8d13-4f0d894780ad.png)

Desktop view does the same thing. I'm not sure what the intention of setting padding on the body element is.

Reply:
![image](https://user-images.githubusercontent.com/109072464/222721322-139eaff6-26ec-47cf-8987-4753429bc5f4.png)